apps/src/templates/manageStudents/MoveStudents.jsx

```diff
@@ -249,13 +244,20 @@ class MoveStudents extends Component {
     });
   };
 
+  isMovableSection = (section) => {
```

Comment:
This name is a little misleading, since it's not a question of whether the _section_ is movable.  I'd use something like `isValidDestinationSection` but I tend to err on the verbose side.